src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivitiesResource.kt

```diff
@@ -310,9 +297,6 @@ class ActivitiesResource(
       ApiResponse(
         responseCode = "200",
         description = "OK",
-        content = [
-          Content(mediaType = "application/json", schema = Schema(implementation = FindActiveActivityIdsResponse::class)),
```

Comment:
Unnecessary, and in this case also inaccurate so it was breaking generated code in a client.